moqt-server/src/modules/handlers/announce_handler.rs

```diff
@@ -1,13 +1,57 @@
 use anyhow::Result;
-use moqt_core::constants::StreamDirection;
-use moqt_core::pubsub_relation_manager_repository::PubSubRelationManagerRepository;
+
 use moqt_core::{
+    constants::StreamDirection,
     messages::control_messages::{
         announce::Announce, announce_error::AnnounceError, announce_ok::AnnounceOk,
     },
-    MOQTClient, SendStreamDispatcherRepository,
+    pubsub_relation_manager_repository::PubSubRelationManagerRepository,
+    SendStreamDispatcherRepository,
 };
 
+use crate::modules::moqt_client::MOQTClient;
+
+async fn forward_announce_to_subscribing_namespace_subscribers(
```

Comment:
I think there is a small problem that the draft has already used `forward` as meaning that sending message to subscriber.

I suggest alternative word `transfer`.
This word has meaning that `to move someone or something from one place to another` and the draft hasn't used this word.
What do yo think about it?

Reply:
Good, I think `transfer` is better than `forward`.